Sources/Plasma/PubUtilLib/plNetCommon/plNetObjectDebugger.cpp

```diff
@@ -265,31 +238,27 @@ void plNetObjectDebugger::ClearAllDebugObjects()
 //
 // write to status log if there's a string match
 //
-void plNetObjectDebugger::LogMsgIfMatch(const char* msg) const
+void plNetObjectDebugger::LogMsgIfMatch(const ST::string& msg) const
 {
-    if (GetNumDebugObjects()==0 || !msg)
+    if (GetNumDebugObjects() == 0 || msg.empty())
         return;
 
     // extract object name from msg, expects '...object:foo,...'
-    std::string tmp = msg;
-    hsStrLower((char*)tmp.c_str());
-    std::string objTag="object";
-    const char* c=strstr(tmp.c_str(), objTag.c_str());
-    if (c && c != tmp.c_str())
+    const auto objTag = ST_LITERAL("object");
+    ST_ssize_t pos = msg.find(objTag, ST::case_insensitive);
```

Comment:
If I read this correctly, we would previously not match `ObJeCt`, but we now do. Minor change of behavior, but I think it's fine.

Reply:
Actually, we would have matched it before too, since the whole message was converted to lower-case before searching.